desktop/src/main/java/pmstation/configuration/Config.java

```diff
@@ -47,7 +47,13 @@ public enum Entry {
         CHECK_LATEST_VERSION("check.latest.version"),
         WINDOW_THEME("window.theme"),
         CSV_LOG_ENABLED("csv.log.enabled"),
-        CSV_LOG_FILE("csv.log.file");
+        CSV_LOG_FILE("csv.log.file"),
+        MQTT_CLIENT_ID("mqtt.client_id"),
```

Comment:
add MQTT_ENABLED akin to CSV_LOG_ENABLED("csv.log.enabled"). Set it to default `false` when config loaded. Edit your conf manually to enable it (since you haven't added it to UI Config). Would be nice to update README.md to state that option.